templates/zerver/register.html

```diff
@@ -124,6 +124,7 @@ <h1>{{ _('Create your account') }}</h1>
                             {% endfor %}
                         {% endif %}
                     {% endif %}
+                    <p id="full_name_help_text">This is how your name will be displayed to others.</p>
```

Comment:
This needs a translation tag for the string.

Reply:
Fixed.